src/proto/streams/prioritize.rs

```diff
@@ -630,6 +654,10 @@ impl Prioritize {
             tracing::trace!(?frame, "dropping");
         }
 
+        // FIXME(nox): Do we need to wake something up when the current send
+        // buffer size was equal to max send buffer size and decreased?
```

Comment:
Yes, since a stream may have been parked waiting for "capacity" while the send buffer was "full". In other places in this file, that is done by calling `assign_connection_capacity()`. It would be wrong to call that unmodified, since more room in the buffer doesn't mean the flow control capacity should be changed. But perhaps a similar `assign_connection_buffer()` that does similar work would be the right call.

Reply:
Maybe calling it with 0 is enough?

Reply:
Well, this comment is kinda outdated. It is in `Prioritize::clear_queue`, which has only 2 callers:
* `Send::send_reset`, which then unconditionally calls `Prioritize::reclaim_all_capacity`;
* `Send::handle_error`, which also unconditionally calls `Prioritize::reclaim_all_capacity`.

Then, `Prioritize::reclaim_all_capacity` unconditionally calls `Prioritize::assign_connection_capacity`.

I can't find the path on which I forgot to add a call to `assign_connection_capacity`.